src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2727,7 +2727,22 @@ private string GenerateCertificateThumbprintParameter()
 
```

Comment:
@KirkMunro, your last commit had 8 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Stream writer tests.Stream Common Parameter Tests.Should prefer -VerboseAction over -Verbose when both are provided
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Write-Stream.Tests.ps1: line 236
236:             $streamData = @(Test-StreamData @parameters *>&1)

```
Stream writer tests.Stream Common Parameter Tests.Should prefer -DebugAction over -Debug when both are provided
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Write-Stream.Tests.ps1: line 250
250:             $streamData = @(Test-StreamData @parameters *>&1)

```
Stream writer tests.Stream Common Parameter Tests.Should be able to ignore all streams except for the Error stream
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Write-Stream.Tests.ps1: line 147
147:                 Write-Progress -Activity 'Warming up' -PercentComplete ([Math]::Round(0 / 6 * 100))

```
Stream writer tests.Stream Common Parameter Tests.Should be able to ignore all streams except for the Warning stream
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Write-Stream.Tests.ps1: line 221
221:             $streamData = @(Test-StreamData @parameters *>&1)

```
Stream writer tests.Stream Common Parameter Tests.Should be able to ignore all streams except for the Verbose stream
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Write-Stream.Tests.ps1: line 221
221:             $streamData = @(Test-StreamData @parameters *>&1)

```

Reply:
These errors are all due to moving the removal of the `ActionPreference.Ignore` restriction to PR #10317, as @daxian-dbw requested. Once that PR is merged, I'll merge those changes into this PR and all Pester errors will go away again.